### PR TITLE
Add more model names to Kissmetrics (UPLOAD-881)

### PR DESCRIPTION
**Pull request by gniezen**
Addresses [UPLOAD-881].

[UPLOAD-881]: https://tidepool.atlassian.net/browse/UPLOAD-881?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: app/package.json [hunk 1]
```diff
@@ -13,6 +13,6 @@
     "drivelist": "11.0.0",
     "keytar": "7.9.0",
     "@ronomon/direct-io": "3.0.1",
-    "usb": "2.5.2"
+    "usb": "2.4.3"
```
Comment (krystophv) [inline app/package.json:16]: Just want to verify that this is an intentional downgrade? It's not obviously attached to the device model number adjustment/additions in a clear way.

**Review by krystophv — APPROVED** [review]
LGTM 👍🏼 

I did have a question about the `usb` version shift, but not holding up approval for it.

File: app/package.json (hunk 1 above)
Comment (gniezen) [reply app/package.json:16]: The most recent `node-usb` update broke things - I notified them (https://github.com/node-usb/node-usb/pull/549) and they fixed it, but they haven’t pushed a new release yet.

**Comment by gniezen** [comment]
@krystophv The changes since your last review are a little more substantial than just removing the test suffix, so can you please have another look?

The submodule change should just be to update to the latest main branch of the tandem driver.

**Pull request merged**